go/libkb/login_state.go

```diff
@@ -461,10 +461,7 @@ func (s *LoginState) resetOrDelete(un string, which string) (err error) {
 			SessionR:    lctx.LocalSession(),
 		}
 		pdpka.PopulateArgs(&arg.Args)
-		res, aerr := s.G().API.Post(arg)
-		if aerr == nil {
-			s.G().Log.Info("%s Result: %+v\n", which, res.AppStatus)
```

Comment:
removed so it doesn't print to stdout we can `Log.Debug` if anyone feels strongly